### PR TITLE
coverage_fixes

### PR DESCRIPTION
**Pull request by reidkwja**

### CODE REVIEW

File: tests/covscripts/coverage_parse_attributes.py [hunk 1]
```diff
@@ -24,18 +24,19 @@
 #
 ###############################################################################
 import json
-import sys
+import os
 
-sys.path.append("../tuna")
-sys.path.append("tuna")
+root_dir = os.path.dirname(
```
Comment (alexandraBara) [inline tests/covscripts/coverage_parse_attributes.py:29]: @cderb is there a better way to get the root  dir in Tuna without having to do this?

Comment (cderb) [reply tests/covscripts/coverage_parse_attributes.py:29]: Maybe by using relative path? "../../../"

**Comment by reidkwja** [comment]
> 



> 

@alexandraBara 

Ahhh....For  that one, install coverage `pip install coverage` ... version == 6.4.2

**Review by alexandraBara — APPROVED** [review]
Paths have been fixed, LGTM

**Review by cderb — APPROVED** [review]

**Pull request merged**